tutorial/01-Introduction-to-Ibis.ipynb

```diff
@@ -1,14 +1,33 @@
 {
```

Comment:
<p>I think we should drop the link here - users will be running these in binder or locally - in either case a link to the next notebook on GitHub won't do the right thing.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-examples/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
oop, just noticed that the link is a relative link, nevermind, this is good as is!